include/curl-session.h

```diff
@@ -78,7 +78,8 @@ class CurlSession : public CurlCallbacks
     }
 
 
-    void addFormFile(const std::string& key, const std::string& filePath, const std::string& mimeType);
+    void addFormFile(const char* key, const char* filePath, const char* mimeType);
```

Comment:
Even if it does not, std::strings are pretty efficient. I do not think this is critical optimization. It just C style interface now. Not C++.